test/sql/secrets/create_secret_storage_backends.test

```diff
@@ -31,6 +31,10 @@ Invalid Input Error: Persistent secrets are disabled. Restart DuckDB and enable
 
 restart
 
+# Enable persistent secrets so we can set a 'secret_directory'
```

Comment:
Likely another case of difference in behavior between the python and c++ test runner.
The C++ runner has access to the internal DBConfig of the database, the python one does not.

When this change is removed, the python runner errors on the next test-statement:
```
test/sql/secrets/create_secret_storage_backends.test:38 Query unexpectedly failed: Invalid Input Error: Persistent secrets are disabled. Restart DuckDB and enable persistent secrets through 'SET allow_persistent_secrets=true'
```